### PR TITLE
[Docs] Replace connection-settings with device-manager

### LINKED ISSUE #560: documentation - device manager

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #560 

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: docs/device-manager.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+PingViewer automatically scans for available devices on serial ports (COM) and UDP connections. If a Ping device is detected, the available connection is shown and the user will be able to choose between all available devices in **Device Manager**.
+
+PingViewer attempts to connect to UDP port 9090 [Ping1D] and 9092 [Ping360] on the host at IP address 192.168.2.2. This is the default IP address of the **Companion computer** that runs on the BlueROV2. The **Companion computer** automatically configures a communication bridge on these ports to a connected sensor.
```
Comment (Williangalvani) [inline docs/device-manager.md]: "ports" instead of "port"?

File: docs/device-manager.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    5- Connects with the configured settings.
+    6- Goes back to **Device Manager** menu.
+    7- Cancel and exit **Manual Connection**.
+
+## Connection Types
+
+- **Serial:** Select the serial port (eg. `COM1` or `ttyUSB0`) and baudrate, that will be updated by the possible baud rates for the sensor.
+- **UDP:** Select the host UDP server IP address and port.
+
+> Note: To connect the application to a Ping device on a remote host (eg. a Raspberry Pi) you can use the pingproxy program in the bluerobotics-ping python package:
```
Comment (Williangalvani) [inline docs/device-manager.md]: I think this only works for Ping1D

File: docs/device-manager.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    1- Shows an animation and the name of the available sensor.
+    2- Describes the kind of the available connection.
+    3- Shows the availability of the connection.
+        **Red**: The connection is not available/reliable.
+        **Green**: The sensor is connected and available for connection.
+    4- Describes the connection type and the connection configuration.
+    5- Allows the user to go to **Manual Connection**.
+    6- Cancel and exit **Device Manager**.
+
```
Comment (Williangalvani) [inline docs/device-manager.md]: I think "manual connection" should be a header too (## Manual Connection).
And maybe "connection types" not, or make it a sub-header

File: docs/device-manager.md (hunk 1 above)
Comment (patrickelectric) [reply docs/device-manager.md]: 9090 and 9092, two ports.

File: docs/device-manager.md (hunk 2 above)
Comment (patrickelectric) [reply docs/device-manager.md]: This should be specified in ping-python repository and not in ping-viewer.

File: docs/device-manager.md (hunk 1 above)
Comment (Williangalvani) [reply docs/device-manager.md]: "to UDP **ports** 9090 [Ping1D] and 9092 [Ping360] "
I would also use parenthesis instead

Comment (patrickelectric) [reply docs/device-manager.md]: Done.

**Comment by patrickelectric** [comment]
> Looks good. Should we wait until closer to release to merge this, so we don't confuse the current users?

@jaxxzer 	^

**Comment by jaxxzer** [comment]
Yes let's wait till this makes stable.

File: docs/device-manager.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    7- Cancel and exit **Manual Connection**.
+
+## Connection Types
+
+- **Serial:** Select the serial port (eg. `COM1` or `ttyUSB0`) and baudrate, that will be updated by the possible baud rates for the sensor.
+- **UDP:** Select the host UDP server IP address and port.
+
+> Note: To connect the application to a Ping device on a remote host (eg. a Raspberry Pi) you can use the pingproxy program in the bluerobotics-ping python package:
+>
+> `sudo pip install bluerobotics-ping`
```
Comment (jaxxzer) [inline docs/device-manager.md]: use install --user, no sudo

**Pull request merged**